friends/src/components/FriendsList.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export class FriendsList extends Component {
```

Comment:
Heads up, no need to make this a class component unless you're dealing with state or life cycle methods.